apps/hash-graph/lib/graph/src/store/fetcher.rs

```diff
@@ -226,6 +226,7 @@ where
                 break;
             }
 
+            dbg!(&ontology_urls);
```

Comment:
I wonder if there is a lint for this.

```suggestion
```

Reply:
Something is weird with the Rust lints for sure